api.bs

```diff
@@ -921,6 +932,120 @@ To <dfn>deduct privacy budget</dfn> given a [=privacy budget key=] |key|,
 1.  Return whether |newValue| is greater than or equal to 0.
 
 
+### Epoch Start Store ### {#s-epoch-start}
+
+An [=epoch=] starts at a randomly-selected time
+for each [=site=].
+This ensures that any bias in aggregates
+that arises from a [=privacy budget=] reaching zero
+is averaged across all the contributions from all [=users=].
+
+The <dfn>epoch start store</dfn> is a [=map=] keyed by [=site=]
+with values that are [=moments=].
+
+To <dfn>get the current epoch</dfn>
+given an [=environment settings object=] |settings|,
+a [=site=] |site|,
+and [=moment=] |now|:
```

Comment:
Do we need to take any special care for the non-monotonically-increasing times throughout these algorithms?

https://w3c.github.io/hr-time/#sec-clocks

Reply:
Opened #123 for that question, which is separable.